packages/itwin/measure-tools/package.json

```diff
@@ -23,6 +23,7 @@
     "rebuild": "npm run clean && npm run build",
     "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
     "lint:fix": "npm run lint -- --fix",
+    "lint:table": "eslint ./**/*.{ts,tsx} 1>&2 -f ./node_modules/@itwin/eslint-plugin/dist/formatters/no-internal-summary-with-table.js",
```

Comment:
this currently creates a csv, by default when you run this, which should be git ignored 
theres this pr which should fix that https://github.com/iTwin/eslint-plugin/pull/78

Reply:
added to git ignore

Reply:
https://www.npmjs.com/package/@itwin/eslint-plugin/v/4.1.1, published

Reply:
i guess dan, plz update to use this version then, but keep the gitignore change